[WIP] Initial contribution for Montreal 2020

anthonydahanne opened the pull request.

idcrosby commented on content/events/2020-montreal/_index.md:
```diff
@@ -0,0 +1,105 @@
+  homepage: https://www.linkedin.com/in/ayrat-khayretdinov-8503b6b
+- name: Frédéric Médery
+  image: organizers/frederic.jpg
+  twitter: https://ca.linkedin.com/in/fredericmedery
```
does Frédéric have a twitter account?

anthonydahanne replied:
just pinged him on slack

fmedery replied:
alias fmedery

idcrosby commented on content/events/2020-montreal/contact.md:
```diff
@@ -0,0 +1,7 @@
+---
+title: Team / Contact
```
Because of the bilingual titles the top bar cuts off (at least for me)
<img width="1280" alt="Screenshot 2020-03-04 at 06 17 17" src="https://user-images.githubusercontent.com/1084449/75874529-e3d99880-5ddf-11ea-8f4c-e9bb2609cfb4.png">

idcrosby replied:
Seems to be a general problem with the site design, if I resize the browser the whole bar disappears :( but I'm no front end dev so 🤷‍♂

anthonydahanne replied:
yeah; that does not look great; I'll probably revert back to single language for titles

anthonydahanne replied:
unless I feel creative with css...

lucperkins replied:
FYI I'm currently working on a fix for this

jfchevrette commented:
Please add me to the organizers list

```yaml
- name: Jean-François Chevrette
  image: (use twitter profile pic: https://pbs.twimg.com/profile_images/810953564751364096/lSRKaRHN_400x400.jpg)
  twitter: https://twitter.com/jfchevrette
  homepage: https://www.linkedin.com/in/jfchevrette/
```

lucperkins commented:
@anthonydahanne @idcrosby In PR #319 I provided a way for projects to have a dropdown menu, as seen in the example [Bouvet Island project](https://kubernetescommunitydays.org/events/2019-bouvet-island/). Just add this to the metadata in `_index.md`:

```yaml
dropdown_menu: true
```

idcrosby commented:
This is great, thanks @lucperkins! So are you gonna come back to Montreal for our event then? 😄

lucperkins commented:
@idcrosby I'd love to! I had an absolute **blast** last time. The only downside was that I didn't get to see the Raptors win it all in the big crowd 😢 But glad they pulled it out after I flew home 💪 I'll see what I can do...

anthonydahanne commented:
> Please add me to the organizers list

done @jfchevrette

anthonydahanne commented:
> @anthonydahanne @idcrosby In PR #319 I provided a way for projects to have a dropdown menu, as seen in the example [Bouvet Island project](https://kubernetescommunitydays.org/events/2019-bouvet-island/). Just add this to the metadata in `_index.md`:
> 
> ```yaml
> dropdown_menu: true
> ```
> 
> This should address the crowding issue.

thank you, I rebased and added the change

juliasimon commented on content/events/2020-montreal/_index.md:
```diff
@@ -0,0 +1,110 @@
+#  twitter: https://twitter.com/cloudnativefdn
+#  bio: Phippy is a simple PHP app, trying to find a home in a cloud native world.
+#  homepage: https://phippy.io
+organizers:
```
Julia Simon
https://twitter.com/JuliaSimon14
https://github.com/juliasimon
https://www.linkedin.com/in/jsimon14/
![Julia_pic](https://user-images.githubusercontent.com/17503776/76340076-5ca18e80-62d1-11ea-8098-61b29330fded.png)

anthonydahanne replied:
added

KTan1226 commented:
Hi @anthonydahanne- Just checking in on this PR. Is it ready for review?

anthonydahanne commented:
hello @KTan1226 - sure it's now ready for review; although the date has been removed, as well as the venue, since they could change...

KTan1226 commented:
Hi @anthonydahanne- thanks for submitting. Is it possible to put a temporary date for now (you can always change later)? Without a date, the website isn't showing up on the site. Additionally, I would recommend that you include a short note about registration and the CFP instead of leaving them blank for now. A good example of the initial page is KCD Portland: https://kubernetescommunitydays.org/events/2020-portland-oregon/. 

In regards to the CFP, I'm happy to get you access to SM Apply. Let me know what you prefer and we can go from there.

Thanks for the great start! I know its tricky getting the page together when the venue and date are TBD but this will be a good page to point folks to for information as it develops.

anthonydahanne commented:
@KTan1226 Hello, I have re added the dates, and made them more realistic.
I'm not sure about your comment about registration/CFP; our pages aren't blank... I've reviewed them a bit, please let me know if they're fine now or if they require some tweaking.
thank you!

anthonydahanne commented:
hello @KTan1226 - everything ok with the PR? can we help if not? (well, we still haven't locked the venue or even the date; but the team shall meet soon and discuss those topics)

KTan1226 commented:
Hi @anthonydahanne! Everything looks good to me. My apologies- I accidentally clicked the revert button. I'll see if I can remove it from the page. Appreciate you checking :)
